test/sql/pg_dump.sql

```diff
@@ -177,11 +178,8 @@ SELECT get_sqlstate('SELECT timescaledb_post_restore()');
 
 drop function get_sqlstate(TEXT);
 
---use a standard dbname because :TEST_DBNAME is different on 9.6 vs 10 & 11
---and dbname is displayed in error
-\c :TEST_DBNAME :ROLE_SUPERUSER
+\c postgres :ROLE_SUPERUSER
 --need to shutdown workers to use db as template
```

Comment:
Can you update or remove this comment? E.g.
```suggestion
--workers must be shut down before using the db as template, which is done above
```